guides/cookbook.md

```diff
@@ -1061,6 +1186,101 @@ class ReadNumericValues
 ```
 </div>
 
+<div class="tabpane Go" markdown="1">
+
+A reader can have various types of numeric values:
+  - Int32
+  - Int64
+  - Uint64
+  - BigInt
+  - Float
+  - Decimal
+
+The following example illustrates how to read different numeric values in a reader:
+```Go
+package main
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/amzn/ion-go/ion"
+)
+
+func main() {
+	int32Value := 2147483646
+	var int64Value int64 = 9223372036854775807
```

Comment:
```suggestion
int64Value := 9223372036854775807
```

Reply:
I need to explicitly define this value as an int64.